src/main/kotlin/me/partlysanestudios/partlysaneskies/features/farming/endoffarmnotifer/Range3d.kt

```diff
@@ -25,6 +25,9 @@ class Range3d(x1: Double, y1: Double, z1: Double, x2: Double, y2: Double, z2: Do
         rangeName = ""
     }
 
+    fun isInRange(point3d: Point3d): Boolean {
```

Comment:
Considering range3d is more general (i thunk it is) we could move it into the render points thingy aswell, and rename the points package